lib/inspec/plugin/v2/registry.rb

```diff
@@ -50,6 +50,12 @@ def path_based_plugin?(name)
       known_plugin?(name.to_sym) && registry[name.to_sym].installation_type == :path
     end
 
+    def profile_bundled_plugin?(name)
+      return unless known_plugin?(name.to_sym)
+
+      registry[name.to_sym].installation_type == :profile_bundled || name.start_with?("profile")
```

Comment:
Unfortunately, the inspec codebase mixes strings and symbols freely. There are some test failures on ruby 2.4-2.6 from `name.start_with?("profile")` because `name` is a `Symbol`, and `Symbol#start_with?` is only defined in Ruby 2.7+. I suggest `name.to_s.start_with?`.